Anlab.Core/Domain/Order.cs

```diff
@@ -52,9 +52,13 @@ public Order()
         public PaymentEvent ApprovedPayment { get; set; }
         
         public string KfsTrackingNumber { get; set; }
+        public string SlothTransactionId { get; set; } //TODO: Change to Guid? ?
 
         public bool Paid { get; set; } = false;
 
+        public bool IsInternalClient { get; set; } = false;
```

Comment:
@jSylvestre Ah i was thinking to replace this with a single field like string PaymentType which would be like status.  values of "CreditCard", "UCDAccount" "IOCAccount" and "Other" perhaps.  Plus it could cover other cases for us later too.

Reply:
@jSylvestre Looking through the rest of the code I guess this would be a big change, so maybe don't worry about this part